src/style.css

```
@@ -0,0 +1,109 @@
+body {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+.item-card {
+  width: 200px;
+  box-sizing: border-box;
+  border-radius: 5px;
+
+  font-family: "Roboto", sans-serif;
+}
+
+.image-card {
+  padding: 31px 20px 41px;
+}
+
+.card-info {
+  padding: 0 15px 15px;
+}
+
+.card-name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin-bottom: 5px;
+}
+
+.card-code {
+  font-size: 11px;
+  line-height: 13px;
+  margin-bottom: 13px;
+}
+
+.card-rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 17.5px;
+  margin-left: 1px;
```

Don't repeat margin property.
```suggestion
  margin: top right bottom left;
```

---

remove repeat of margin property